### DISCUSSION TITLE
Welcome to Cleaver Discussions!

### DISCUSSION THREAD

**Discussion by jessdtate** [SCIInstitute/Cleaver2#117] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zahra-Soltani-Aerospace** [comment]
Hi everyone,

I am using Clever in 3D Slicer for generating tetrahedral mesh. The mesh generated is by default based on the 4-node linear tetrahedral element (C3D4). How can I change the element type to the 10-node quadratic tetrahedral element (C3D10)?

Thanks,
Zahra

**Comment by jessdtate** [comment]
Higher order elements aren't supported in Cleaver at this time.  However, the C3D4 mesh can be converted to C3D10 by adding nodes at the midpoints of the edges.  Other software can to this, but FEBio studio (https://github.com/febiosoftware/FEBioStudio) is one that is geared toward biomachanics simulation, and is also produced at SCI

**Reply by Zahra-Soltani-Aerospace** [reply]
Thanks @jessdtate for your reply. 

Yes, adding midpoints to C3D4 elements is the easiest way, but it's not optimized and efficient computationally. Because however we will have C3D10 (that means higher numbers of nodes/DOFs), still the mesh generation is based on the linear elements that need to be finer to fit the curved boundary surfaces.

I will look at other alternatives, such as Gmsh and FEBio that you mentioned.

Thanks,
Zahra
